src/main/java/appeng/integration/modules/theoneprobe/part/StorageMonitorInfoProvider.java

```diff
@@ -48,10 +48,11 @@ public void addProbeInfo( IPart part, ProbeMode mode, IProbeInfo probeInfo, Enti
 			final IAEStack<?> displayed = monitor.getDisplayed();
 			final boolean isLocked = monitor.isLocked();
 
+			// TODO: generalize
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)